content/webapp/components/Discussion/Discussion.tsx

```diff
@@ -32,7 +32,7 @@ const Discussion: FunctionComponent<Props> = ({ title, text }: Props) => {
     if (isActive) {
       setTextToShow(text);
     } else {
-      setTextToShow(firstPartOfText);
+      setTextToShow(firstPartOfText as [prismicT.RTNode, ...prismicT.RTNode[]]);
```

Comment:
How does `[prismicT.RTNode, ...prismicT.RTNode[]]` differ from `prismicT.RTNode[]`? Does it make the first one required and any others optional?

Reply:
Honestly not sure, it's a type signature from Prismic and it's what it took to make the type checker happy. 🤷‍♀️